src/pkg/bundle/dev.go

```diff
@@ -47,8 +49,13 @@ func (b *Bundle) CreateZarfPkgs() {
 				}
 			}
 			// create local zarf package if it doesn't exist
-			if !packageFound {
-				os.Args = []string{"zarf", "package", "create", pkgDir, "--confirm", "-o", pkgDir, "--skip-sbom"}
+			if !packageFound || b.cfg.DevDeployOpts.ForceCreate {
+				if len(b.cfg.DevDeployOpts.Flavor) != 0 {
+					b.setPackageFlavor(&pkg)
+					os.Args = []string{"zarf", "package", "create", pkgDir, "--confirm", "-o", pkgDir, "--skip-sbom", "--flavor", pkg.Flavor}
```

Comment:
I think you may be able to refactor some of this, looks like `--flavor` can be `""`, so `--flavor=""` is equivalent to omitting the flavor flag. (note that I tested this locally but my knowledge of flavors is limited, may be more nuance here, recommend testing more)

Reply:
Have you seen a folder called "refs" randomly be created when using flavors?

Reply:
so when we run populateFlavorMap() in src/cmd/dev.go we check for `--flavor ""` in which case we don't populate the flavor map

Reply:
gotcha, makes sense